pkg/sdk/testint/network_rule_gen_integration_test.go

```diff
@@ -182,4 +182,14 @@ func TestInt_NetworkRulesShowByID(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, id2, e2.ID())
 	})
+
+	t.Run("show by id: check fields", func(t *testing.T) {
+		name := random.AlphaN(4)
+		id1 := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
```

Comment:
check can be addded to assertValuesAndComment, no need for a separate test

Reply:
Done.